src/include/transaction/deferred_action_manager.h

```diff
@@ -62,17 +64,28 @@ class DeferredActionManager {
    * Clear the queue and apply as many actions as possible
    * @return numbers of deferred actions processed
    */
-  uint32_t Process(transaction::timestamp_t oldest_txn) {
+  uint32_t Process() {
```

Comment:
Ideally most of these functions should be in `deferred_action_manager.cpp`. Currently the .cpp doesn't really have much in it. Might as well get rid of it and move those functions here too if it's too much of a hassle to move all the implementations to the `.cpp`